src/argilla_server/contexts/questions.py

```diff
@@ -50,6 +50,7 @@ def _validate_settings_type(settings: QuestionSettings, settings_update: Questio
 
 
 def _validate_label_options(settings: LabelSelectionQuestionSettings, settings_update: LabelSelectionSettingsUpdate):
+    # TODO: Validate visible_options on update
```

Comment:
What kind of validation?

Reply:
Same as for the question creation: to be sure the limits for `visible_options`